composer.json

```diff
@@ -36,11 +36,20 @@
         "squizlabs/php_codesniffer": "~2.0",
         "vlucas/phpdotenv": "^2.0 | ^3.0 | ^4.0 | ^5.0",
         "symfony/process": ">=2.7 <6.0",
+        "codeception/module-apc": "*@dev",
         "codeception/module-asserts": "*@dev",
         "codeception/module-cli": "*@dev",
         "codeception/module-db": "*@dev",
         "codeception/module-filesystem": "*@dev",
+        "codeception/module-ftp": "*@dev",
+        "codeception/module-memcache": "*@dev",
+        "codeception/module-mongodb": "*@dev",
         "codeception/module-phpbrowser": "*@dev",
+        "codeception/module-redis": "*@dev",
+        "codeception/module-rest": "*@dev",
+        "codeception/module-sequence": "*@dev",
+        "codeception/module-soap": "*@dev",
+        "codeception/module-webdriver": "*@dev",
```

Comment:
If this is for Docker only, please add `composer require` commands to Dockerfile.

Reply:
Well there is no "docker only" either it's needed to run the tests and it should be in `require-dev` or it should not be anywhere. requiring packages only in the dockerfile makes no sense imho?

Reply:
Dockerfile is used to build images that are published at https://hub.docker.com/r/codeception/codeception
Some people may be using it to run their tests, so the docker image provides the same modules as phar file.

Modules that are currently listed in `require-dev` section are the only modules that are required to run Codeception test suite. The fewer, the better.

Reply:
alright, that makes sense, already had a hunch the dockerfile had some other purpose than just local development. If I had some spare time during the hollidays I'll update the PR